front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/plugin/Plugin.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.front50.model.plugin;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.hash.Hashing;
+import com.netflix.spinnaker.front50.model.Timestamped;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+public class Plugin implements Timestamped {
```

Comment:
what is the story for the location of the plugin binary?

I like that it isn't directly in here, but should this include a pointer URI or something to the contents?

Reply:
I think that is an open question.  I'm going to hold off on adding that for the time being.

Reply:
Expanding on that - I believe the current thought was to specify repositories via kork.

Reply:
👍 seems legit to start with